docs/anonymous-statistics.md

```diff
@@ -70,6 +70,11 @@ events is:
 Furthermore, the FATAL event sends the Netdata process & thread name, along with the source code function, source code
 filename and source code line number of the fatal error.
 
+Starting with v1.21 we additionally collect information about:
+- Failures to build the dependencies required to be able to use Cloud features.
+- Unavailability of Cloud features in Agent.
+- Failures to connect to the Cloud in case the Agent has been claimed (you register Agent in Cloud). This includes error code informing us about the reason why the connection failed.
+
```

Comment:
```suggestion
Starting with v1.21, we additionally collect information about:
- Failures to build the dependencies required to use Cloud features.
- Unavailability of Cloud features in an agent.
- Failures to connect to the Cloud in case the agent has been [claimed](../../claim/README.md). This includes error codes to inform the Netdata team about the reason why the connection failed.
```